### ISSUE TITLE
Is it possible to run a supervised model with transformer backbone to compare it with SSL methods?

### ISSUE THREAD

**Issue by Vijayraven95** [vturrisi/solo-learn#234]

**Comment by DonkeyShot21** [comment]
It is not possible by default but you can remove this detach: https://github.com/vturrisi/solo-learn/blob/f25dc1fccb3c3ed2dbd848138adc2d5594f671a4/solo/methods/base.py#L492 and zero out the SSL loss. Then ofc you will need to do hyperparameter tuning and probably change the data augmentations slightly to make transformers work well. I suggest you take a look at [timm](https://github.com/rwightman/pytorch-image-models).
